DUI3-DX/Connectors/Revit/Speckle.Connectors.RevitShared/Bindings/SendBinding.cs

```diff
@@ -14,7 +14,7 @@
 
 namespace Speckle.Connectors.Revit.Bindings;
 
-internal class SendBinding : RevitBaseBinding, ICancelable
+internal class SendBinding : RevitBaseBinding, ICancelable, ISendBinding
```

Comment:
Not sure we need these specific interfaces but it's not an issue atm

Reply:
I don't think we "need" them as it worked before, but I thought we should have it for consistency.